### ISSUE TITLE
import method calls are incorrectly marked as unsupported

### ISSUE THREAD

**Issue by LJNeon** [mysticatea/eslint-plugin-node#210]
**Environment**

 * **Package Versions:**
   * eslint v6.8.0
   * eslint-plugin-node v11.0.0
 * **Node Version:** v13.11.0
 * **npm Version:** v6.13.7
 * **Parser:** babel-eslint v10.1.0 (for import meta syntax)

**Configuration:**

<details>
<summary>.eslintrc.yml</summary>

```yml
extends: ["eslint:recommended", "plugin:node/recommended"]

rules: {
  # ...
}

parser: "babel-eslint"

parserOptions: {
  ecmaVersion: 2020,
  sourceType: "module"
}
```

</details>

`package.json` has both `"type": "module"` and `"engines": {"node": ">=13.11.0"}` set.

**Command used to run ESLint:** `eslint src --ext .js`

**What did you expect to happen?**
ESLint wouldn't error, as `import()` is supported syntax that I need to use.

**What actually happened?**
```bash
/home/user/project/src/index.js
  46:15  error  'import()' expressions are not supported yet  node/no-unsupported-features/es-syntax
  48:27  error  'import()' expressions are not supported yet  node/no-unsupported-features/es-syntax
  52:35  error  'import()' expressions are not supported yet  node/no-unsupported-features/es-syntax
  53:37  error  'import()' expressions are not supported yet  node/no-unsupported-features/es-syntax
```
I removed private info from the file path but that response is otherwise untouched.

**Comment by tafelnl** [comment]
I have the same issue with about the same environment.

As mentioned [here](https://github.com/mysticatea/eslint-plugin-node/issues/140#issuecomment-619009314), I think it would be nice to be able to ignore this warning.

**Comment by LJNeon** [comment]
Actually, I'm only using babel because the default parser doesn't support the syntax I want yet. This could be determined as supportable purely from the Node.js version, so I doubt it's related to using babel in my case.

**Comment by tafelnl** [comment]
@LJNeon Yes that could be.

Whatever the case, I think it should be included in here: https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features/es-syntax.md#ignores

That way we can decide for ourselves if we want an error for using import statements over require statements.

**Comment by LJNeon** [comment]
It seems like it's been several months since any issue got a response, is the package abandoned or is there some other reason @mysticatea ?

**Comment by mysticatea** [comment]
I'm sorry. I need time to read English (because my English skill is poor), so I need large energy to answer questions.

About this issue, it works correctly. ES Modules are not stable yet in Node.js. Use `ignores` option to ignore this error.

**Comment by LJNeon** [comment]
I think there's a clear distinction between "unsupported" and "experimental". It'd be nice if something was done to separate the two, instead of lumping them together.

**Comment by LJNeon** [comment]
But at that point we've gone somewhat off-topic, so I'm going to close this and open a more specific issue.
